op-chain-ops/genesis/layer_two_test.go

```diff
@@ -84,6 +93,56 @@ func testBuildL2Genesis(t *testing.T, config *genesis.DeployConfig) *core.Genesi
 	return gen
 }
 
+// testL1Block tests that the state is set correctly in the L1Block predeploy
+func testL1Block(t *testing.T, caller bind.ContractCaller, config *genesis.DeployConfig, block *types.Block) {
+	contract, err := bindings.NewL1BlockCaller(predeploys.L1BlockAddr, caller)
+	require.NoError(t, err)
+
+	number, err := contract.Number(&bind.CallOpts{})
+	require.NoError(t, err)
+	require.Equal(t, block.Number().Uint64(), number)
+
+	timestamp, err := contract.Timestamp(&bind.CallOpts{})
+	require.NoError(t, err)
+	require.Equal(t, block.Time(), timestamp)
+
+	basefee, err := contract.Basefee(&bind.CallOpts{})
+	require.NoError(t, err)
+	require.Equal(t, block.BaseFee(), basefee)
+
+	hash, err := contract.Hash(&bind.CallOpts{})
+	require.NoError(t, err)
+	require.Equal(t, block.Hash(), common.Hash(hash))
+
+	sequenceNumber, err := contract.SequenceNumber(&bind.CallOpts{})
+	require.NoError(t, err)
+	require.Equal(t, uint64(0), sequenceNumber)
+
+	blobBaseFeeScalar, err := contract.BlobBaseFeeScalar(&bind.CallOpts{})
+	require.NoError(t, err)
+	require.Equal(t, config.GasPriceOracleBlobBaseFeeScalar, blobBaseFeeScalar)
+
+	baseFeeScalar, err := contract.BaseFeeScalar(&bind.CallOpts{})
+	require.NoError(t, err)
+	require.Equal(t, config.GasPriceOracleBaseFeeScalar, baseFeeScalar)
+
+	batcherHeader, err := contract.BatcherHash(&bind.CallOpts{})
+	require.NoError(t, err)
+	require.Equal(t, eth.AddressAsLeftPaddedHash(config.BatchSenderAddress), common.Hash(batcherHeader))
+
+	l1FeeOverhead, err := contract.L1FeeOverhead(&bind.CallOpts{})
+	require.NoError(t, err)
+	require.Equal(t, config.GasPriceOracleOverhead, l1FeeOverhead.Uint64())
+
+	l1FeeScalar, err := contract.L1FeeScalar(&bind.CallOpts{})
+	require.NoError(t, err)
+	require.Equal(t, config.GasPriceOracleScalar, l1FeeScalar.Uint64())
+
+	blobBaseFee, err := contract.BlobBaseFee(&bind.CallOpts{})
+	require.NoError(t, err)
+	require.Equal(t, big.NewInt(1), blobBaseFee)
+}
```

Comment:
the `big.NewInt(1)` should maybe be replaced by calculating the blob basefee from the excess-blobgas of the L1 block. Or assert that excess-blob-gas is 0, to sanity check.